packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -360,11 +352,12 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
     onFetchAvatarPersonaData,
     onFetchParticipantMenuItems,
     options,
-    formFactor = 'desktop',
-    /* @conditional-compile-remove(rooms) */
-    roleHint
+    formFactor = 'desktop'
   } = props;
 
+  /* @conditional-compile-remove(rooms) */
+  const roleHint = adapter.getState().roleHint;
```

Comment:
Is there a future work item to move roleHint to selectors?

Reply:
Oh this is getting direct from adapter state because it's being used in the permissions prompt not a component